app/scripts/modules/core/src/pipeline/config/stages/stage.module.js

```diff
@@ -197,6 +202,20 @@ module.exports = angular.module('spinnaker.core.pipeline.config.stage', [
       }
     }
 
+    function stageExcludesAllAccounts(stageType, accounts) {
+      if (stageType.excludedCloudProviders) {
+        const anyNotExcluded = some(accounts, account => {
+          return some(stageType.excludedCloudProviders, excluded => (
```

Comment:
Could make this `stageType.excludedCloudProviders || []` and remove the entire enclosing `if { ... } else { ... }`.

Reply:
done